test/source/tests/compose.ts

```diff
@@ -844,7 +856,7 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
       const text = 'This message is encrypted with 2 keys of flowcrypt.compatibility';
       const subject = `Test Sending Multi-Encrypted Message With Test Text ${Util.lousyRandom()}`;
       const composePage = await ComposePageRecipe.openStandalone(t, browser, 'compatibility');
-      await ComposePageRecipe.fillMsg(composePage, { to: 'flowcrypt.compatibility@gmail.com' }, subject, { sign: true, encrypt: true });
```

Comment:
well, the following line used to fill the body (prepending to existing text if I remember correctly).
```
await composePage.waitAndType('@input-body', text, { delay: 1 });
```
I guess since you moved the body text to a parameter of fillMsg, we should get rid of `waitAndType('@input-body'` as well as `read('@input-body')` here.

Reply:
I changed the logic in `fillMsg()` method so if the nullable or empty `body` param passed, it will fall back to fill the same way as before: https://github.com/FlowCrypt/flowcrypt-browser/pull/4055/files#diff-3f9f08c331d96dfb3de5f73f315d944f2272176b514691eb0c491b2b7b62cd46R72

Would it be clearer for reading if I replace `''` with `undefined`?

Reply:
I'd be happy if you replace `''` with actual values from the tests where we used to `waitAndType` -- there is no need for this separate statement as long as we have the paramater, as I see it. Prepending the text is also not required. Tests mostly search for the substring as I understand.

Reply:
Done 👍

Reply:
is it possible to pass `text` variable instead of `undefined` in this test and remove `await composePage.waitAndType('@input-body', text, { delay: 1 }); ...`?
Will it break the test?

Reply:
Right, sorry I didn't get that from the initial comment. Done 👍